src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.CodeDom.Compiler;
 using System.Collections.Concurrent;
```

Comment:
Is this using necessary?

Reply:
Yes, I was surprised too, but GeneratedCodeAttribute is defined in it.